src/features/Students/StudentsMetrics/index.jsx

```diff
@@ -39,15 +45,15 @@ const StudentsMetrics = () => {
             title="New students registered"
           />
           <Card.Section>
-            <div className="card-number">{stateMetrics.newStudentsRegistered}</div>
+            <div className="card-number">{stateStudentsMetrics.numberOfEnrollments}</div>
```

Comment:
Not sure about to use a `div` for display data, maybe a `p` or `span` tag could be more appropriate.